helm-charts/literacy-data-api/values.yaml

```diff
@@ -0,0 +1,78 @@
+# Default values for literacy-data-api.
+# This is a YAML-formatted file.
+
+import-values:
+  child: service
+  parent: myimports
+
+replicaCount: 3
+
+tags:
+  memcached: true
+
+image:
+  repository: etondreau/literacy-data-api
+  pullPolicy: IfNotPresent
+  env:
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: "/app/var/secrets/keys/bigquery-serviceaccount.json"
+    - name: MEMCACHE_DNS
+      value: "lit-data-memcached"
+    - name: MEMCACHE_PORT
+      value: "11211"
+  volumeMounts:
+    - name: keys
+      readOnly: true
+      mountPath: "/app/var/secrets/keys"
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: "api"
+fullnameOverride: ""
+
+volumes:
+  - name: keys
+    secret:
+      secretName: bigquery-serviceaccount
+
+serviceAccount:
+  create: false
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+
+
+securityContext: {}
+
+service:
+  type: ClusterIP
+  targetPort: 3000
+protocol: TCP
```

Comment:
it's not clear to me what these (3) values are, given they're at root level. Maybe they can be nested so I know what we're referring to? I guess it's the pods themselves as defined in the deployment?

Reply:
I nested those three values under `service` and updated `deployment.yaml` and `service.yaml`